src/TBuffer.cpp

```diff
@@ -2074,19 +2074,37 @@ void TBuffer::append(const QString& text, int sub_start, int sub_end, TChar form
         promptBuffer << false;
         last = 0;
     }
-    bool firstChar = (lineBuffer.back().isEmpty());
+
+    // Whethere this is the first character in the line
+    bool firstChar = (lineBuffer.back().size() == 0);
```

Comment:
:-1: The previous code (using the `isEmpty()`) is held by us (the Mudlet Core Devs) to be a clearer expression of intent than a `size() == 0` - please could you revert this particular line... :pleading_face:

Reply:
I've changed all instance of ".size() == 0" to ".isEmpty()" (there was 3)